Jenkinsfile_k8s

```diff
@@ -1,3 +1,19 @@
+// The node to be used depends on the build type: this function executes the pipeline code block provided as "body"
+// into the correct node type based on the provided arguments
+def withPackerNode(String packer_template, String compute_type, String cpu_architecture, Closure body) {
+  // Build ARM64 CPU Docker images on a native machine (faster than using the local qemu)
+  if (cpu_architecture == 'arm64' && compute_type == 'docker') {
+    node('linux-arm64-docker') {
+      // New agent workspace specified as scripted requires an explicit checkout (compared to declarative)
```

Comment:
Maybe add an echo/log indicating we're using this specific node when arriving here?

Reply:
There already are logs related to the type of agent:

```
12:41:39  Running on [EC2 (aws-us-east-2) - AMD64 Ubuntu 20.04 (i-07891f23c03b73202)](https://infra.ci.jenkins.io/computer/EC2%20%28aws%2Dus%2Deast%2D2%29%20%2D%20AMD64%20Ubuntu%2020.04%20%28i%2D07891f23c03b73202%29/) in /home/jenkins/workspace/infra-tools_packer-images_PR-312
```

```
12:45:03  Running on [EC2 (aws-us-east-2) - ARM64 Ubuntu 20.04 (i-0f66751410e528f00)](https://infra.ci.jenkins.io/computer/EC2%20%28aws%2Dus%2Deast%2D2%29%20%2D%20ARM64%20Ubuntu%2020.04%20%28i%2D0f66751410e528f00%29/) in /home/jenkins/workspace/infra-tools_packer-images_PR-312
```


Also, we can use the "Pipeline Steps" section of each builds, where there is detail of each step in a hierachical view: https://infra.ci.jenkins.io/job/infra-tools/job/packer-images/job/PR-312/1/flowGraphTable/

<img width="1692" alt="Capture d’écran 2022-08-08 à 14 48 38" src="https://user-images.githubusercontent.com/1522731/183421818-b054ac15-62e4-4bcb-9739-566677825d7d.png">